.taskcluster.yml

```diff
@@ -38,17 +38,16 @@ tasks:
             - push
       payload:
         maxRunTime: 3600
-        image: python:3-alpine
+        image: python:3
```

Comment:
What was the problem with alpine?

Reply:
It needs gcc+build-base+python-dev to build `typed-ast` that is a dependency of some hooks